.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,48 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `docs` label][docs-issues] (including
+        closed issues) before you continue.
+
+
+        [docs-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Adocs+is%3Aopen
```

Interesting :thinking: Am I correct that the two extra newlines (line 12 and 13) are necessary to create one newlines in the resulting string and by extension one real newline for the rendered MarkDown? If so, I believe using `|-` will allow us to only have one newline. And due to how MarkDown is rendered (single newlines are ignored) it should still work.

```suggestion
      value: |-
        Before opening a new issue, make sure it isn't covered by an existing issue.
        Please search for [issues with the `docs` label][docs-issues] (including
        closed issues) before you continue.

        [docs-issues]:
        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Adocs+is%3Aopen
```

(this also applies to other instances where there are two extra newlines)

---

That will render to this:
> Before opening a new issue, make sure it isn't covered by an existing issue.
Please search for [issues with the `docs` label][docs-issues] (including
closed issues) before you continue.

As you can see, newlines are still created and this will prevent the text from spanning the whole line. The only way to fix this would be to have the entire text on one line in the YAML, which wouldn't look too great. I've only chosen `>-` over `|-` for these formatting reasons. If you're okay with it looking [like this](https://gist.github.com/sachinraja/a3037e8a16ef0dd352ec4d43fce427db), then we can go ahead and use `|-`.

---

Had to try it by myself to be convinced, it seems you're right :disappointed:

I do prefer the shorter lines (even though we don't (yet) use that in our `.md` files) so I'm happy with this - in general at least.